pkgs/dart_services/analysis_options.yaml

```diff
@@ -22,3 +22,6 @@ linter:
     - prefer_final_in_for_each
     - prefer_final_locals
     - sort_pub_dependencies
+
+formatter:
+  trailing_commas: preserve
```

Comment:
Do we need to override the default formatter behavior?

Reply:
I was trying to minimize diff here but don't really know that this is helping. I can revert this and re-format to see.

Reply:
I think we are already using the latest formatter with the defaults, so this is probably adding some extra noise.